packages/pouchdb-replication/src/backoff.js

```diff
@@ -15,14 +15,19 @@ function backOff(opts, returnValue, error, callback) {
   if (returnValue.state === 'active' || returnValue.state === 'pending') {
     returnValue.emit('paused', error);
     returnValue.state = 'stopped';
-    returnValue.once('active', function () {
+    function backoffTimeSet() {
       opts.current_back_off = STARTING_BACK_OFF;
-    });
+    }
+    function removeBackOffTimeSet() {
+      returnValue.removeListener('active', backoffTimeSet);
+    }
+    returnValue.once('paused', removeBackOffTimeSet);
+    returnValue.once('active', backoffTimeSet);
```

Comment:
It shouldn't as there is an emit 'paused' almost immediately before.  This clears out both listeners before the new one is added.